content/en/docs/components/pipelines/sdk/connect-api.md

```diff
@@ -204,13 +205,12 @@ Cross-namespace access can be achieved in two ways:
   Specifically, the owner of the `namespace-2` uses Kubeflow UI "Manage contributors" page. In the "Contributors to your namespace" 
   textbox he adds email address associated with the `namespace-1`.
 
-##### How Multi-User mode in-cluster authentication works
+#### How Multi-User mode in-cluster authentication works
 
-Authentication uses ServiceAccountToken 
-[projection](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection). 
-Simply put, the token is first being injected into a Pod (e.g. Jupyter notebook's). 
-Then Kubeflow Pipelines SDK uses this token to authorize against Kubeflow Pipelines API.
+When using Kubeflow Pipelines SDK in the same cluster, it authenticates as default-editor in your namespace using ServiceAccountToken 
```

Comment:
"When using Kubeflow Pipelines SDK in the same cluster, [it] authenticates as default-editor.."

Could we be more explicit about the situation and what "it" refers to? For example:

"When multiple users employ the Kubeflow Pipelines SDK in the same cluster, the [service/SDK] authenticates the [namespace/role/user?] as default editor..."